data/colors/vars/global_light.ts

```diff
@@ -40,7 +40,7 @@ export default {
   },
   success: {
     fg: get('scale.green.5'),
-    emphasis: get('scale.green.4'),
+    emphasis: '#1f883d',
```

Comment:
Is this on purpose? Currently we use `#2c8747` in the [feature flag](https://github.com/github/github/blob/440fea807e9348e719371a0545a9b2a6cb122608/app/assets/stylesheets/bundles/color-modes-contrast/index.scss#LL13C31-L13C38).

Or is it to match the output from the new `tokens/`

```
       mix: {
          color: '{base.color.green.5}',
          weight: 0.75,
        },
```

Reply:
Its to match the output.